### PR TITLE
Add openshift-adp jobs for stable branch oadp-1.0

### PR DESCRIPTION
**Pull request by weshayutin**
OADP now needs prow ci to execute on master and stable branch oadp-1.0.

### CODE REVIEW

**Comment by weshayutin** [comment]
@hhpatel14 @deepakraj1997 FYI

**Review by dymurray — APPROVED** [review]

**Comment by rayfordj** [comment]
@weshayutin , while it does LGTM, I noticed it's missing `cli: latest` lines that are present in existing file(s).

Example:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.7.yaml#L79

Should `cli: latest` be included?

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.0__4.7.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test-oadp-operator
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 512Mi
+    workflow: optional-operators-ci-azure
+zz_generated_metadata:
+  branch: master
```
Comment (rayfordj) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.0__4.7.yaml:163]: Like https://github.com/openshift/release/pull/27668/files#diff-60fc118d92d4e178610b009a1d62ab4db3d3c705dd9dba56b3b8ced85a40b2caR158 , shouldn't this be `oadp-1.0`?

**Review by rayfordj — CHANGES_REQUESTED** [review]

**Pull request closed without merging**